example/whoIs/whois.go

```diff
@@ -0,0 +1,124 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"time"
+
+	"github.com/absmach/bacnet/pkg/bacnet"
```

Comment:
The package name seems odd. `bacnet/pkg/bacnet`. Why not move bacnet to the root of the project as this repository is bacnet package